BricksAndHearts.UnitTests/ControllerTests/Admin/AdminControllerTestsBase.cs

```diff
@@ -1,8 +1,11 @@
 ﻿using BricksAndHearts.Controllers;
+using BricksAndHearts.Services;
+using FakeItEasy;
 
 namespace BricksAndHearts.UnitTests.ControllerTests.Admin;
 
 public class AdminControllerTestsBase: ControllerTestsBase
 {
-    protected readonly AdminController _underTest = new(null!, null!, null!);
+    public static readonly IAdminService adminService = A.Fake<IAdminService>();

```

Comment:
I think this should be `protected readonly`, let me know if this doesn't work or if you would like me to clarify the reasoning 🙂

Reply:
I can make it protected rather than public, but if it isn't declared as static then it can't be accessed in the static context of the next line (creating _underTest, an instance of the AdminController). There might be a way around this, but I'm not sure how complicated it would be - I tried adapting the set-up from the start of the normal AdminController and there wasn't an immediately obvious solution - and also not sure whether the static-ness was part of the problem you were commenting on or not. So for now I've changed it to 'protected static readonly'.

Reply:
Having a `static` fake admin service could lead to problems as it will be reused for all the tests.

Can you move the setup to the constructor instead, creating a new fake admin service and `_underTest` controller in there. It should get run before each test.

Reply:
(and same for landlord tests)

Reply:
If it's being run before each test, is it therefore ok for the fake admin service to be static (as in, it would be a static service with a fresh copy being made each time it was needed)?